news/2025/2025-06-27-osucatch-world-cup-2025-round-of-16-recap.md

```diff
@@ -0,0 +1,172 @@
+
+Given all this, no one truly expected Sweden to be able to compete against the United States in this matchup. So it was to everyone's surprise that they went and won their first pick, [NM1](https://osu.ppy.sh/beatmapsets/2388183#fruits/5168115), with a double FC from ::{ flag=SE }:: [amni sd](https://osu.ppy.sh/users/16669841) and ::{ flag=SE }:: [Shameimaru](https://osu.ppy.sh/users/20340480). With that, the United States' perfect record was broken, with their flawless run unexpectedly coming to an end in this Round of 16.
+
+The United States losing a point to Sweden, while unexpected, was also not something too shocking. With how good players have got, even lower-seeded teams are now able to compete and FC maps, leaving top teams with almost no margin for error. The United States brushed that off as a minor misstep and quickly refocused on their match, comfortably winning their next pick on [HR3](https://osu.ppy.sh/beatmapsets/2388197#fruits/5168156) with a second double FC in a row from ::{ flag=US }:: [Lexii](https://osu.ppy.sh/users/7226149) and ::{ flag=US }:: [Secre](https://osu.ppy.sh/users/2306637).
+
+Sweden tried again on their [MM1](https://osu.ppy.sh/beatmapsets/2388341#fruits/5168359) pick, but let the first breakpoint of the match slip away into the United States' favour. They were now down 2 points on their opponent's pick, and the match was going back to what one would expect of this matchup, a mostly one-sided victory from the United States. However, no one could have predicted what happened next.
+
+The United States selected [HD2](https://osu.ppy.sh/beatmapsets/2388193#fruits/5168151), the low AR Hidden pick, knowing what they're able to do on this map. ::{ flag=US }:: [Lexii](https://osu.ppy.sh/users/7226149) managed to get his third FC in a row and ::{ flag=US }:: [Dahcreeper](https://osu.ppy.sh/users/6926006) got his second, but Sweden answered back with a double FC of their own from ::{ flag=SE }:: [amni sd](https://osu.ppy.sh/users/16669841) and ::{ flag=SE }:: [Setomi79](https://osu.ppy.sh/users/2205929) to win the point. Winning a single point against the champions was something, but a second one, and on their own pick? This match was getting intense, and Sweden were performing to the very best of their abilities.
+
+And it didn't stop there, with Sweden following up with mind-blowing 3-way FC on [NM2](https://osu.ppy.sh/beatmapsets/2388185#fruits/5168126) to win their own pick. It had been years since the last time the United States were contested like this so early in the bracket. However, champions don't become champions without strong mental resilience, and they went on to win the next 4 picks in a row, to close out the match 6–3. Even if they couldn't keep taking points from them, Sweden could be proud of their performance, as they gave the United States a battle that they're sure to remember for a while.
```

Comment:
```suggestion
And it didn't stop there, with Sweden following up with a mind-blowing 3-way FC on [NM2](https://osu.ppy.sh/beatmapsets/2388185#fruits/5168126) to win their own pick. It had been years since the last time the United States were contested like this so early in the bracket. However, champions don't become champions without strong mental resilience, and they went on to win the next 4 picks in a row, to close out the match 6–3. Even if they couldn't keep taking points from them, Sweden could be proud of their performance, as they gave the United States a battle that they're sure to remember for a while.
```